### ISSUE TITLE
Corruption of vanilla textures in 1.7.10

### ISSUE THREAD

**Issue by gcewing** [CoFH/Feedback#1078]
Textures of some vanilla blocks are getting scrambled for me when using the following combination of versions (the latest ones at time of writing). No other mods are installed.

forge-1.7.10-10.13.4.1614-1.7.10
CodeChickenLib-1.7.10-1.1.3.138-universal.jar
CodeChickenCore-1.7.10-1.0.7.47-universal.jar
CoFHCore-[1.7.10]3.1.1-324.jar
ThermalExpansion-[1.7.10]4.1.2-240.jar
ThermalFoundation-[1.7.10]1.2.3-112.jar

The attached screenshot shows an example of the problem -- all the vanilla wool block textures have been replaced by random textures from Thermal Expansion.

I've tried various other combinations of versions, all of which show the same kind of problem, although which textures are affected seems to be random.

This is on MacOSX 10.6.8 using the following Java version:
java version "1.7.0_25"
Java(TM) SE Runtime Environment (build 1.7.0_25-b15)
Java HotSpot(TM) 64-Bit Server VM (build 23.25-b01, mixed mode)
(It nags me that I should be using Java 8, but it runs anyway, so it can't be using any features that require Java 8.)

![corruptedwooltextures](https://cloud.githubusercontent.com/assets/1684298/13195494/4940c7b8-d818-11e5-85b9-52a2a34b6d42.png)

**Comment by KingLemming** [comment]
This is unique to your hardware and setup. I suggest installing Java 8 like it tells you to, making sure that you have enough memory allocated, and trying again.

Seriously, we don't get reports of this. It's not something in the mod, but something in Minecraft. If it were us, this would happen 100% every time to everyone. That's the nature of a bug like this if it were in the mod.

**Comment by skyboy** [comment]
Someone else was reporting the same issue some time ago (it's not us, even though the textures are conspicuous). You can search the tracker to see what their resolution was, if it was on the tracker and not in irc.

**Comment by gcewing** [comment]
I found two other issues that look very similar, both on MacOSX: https://github.com/CoFH/Feedback/issues/624 and https://github.com/CoFH/Feedback/issues/906, so it's not just me.

Java 8 doesn't work with Snow Leopard, so I can't try it in my main environment. I tried it with Mavericks, but then the problem doesn't occur for me with any java version. The problem doesn't seem to be restricted to Snow Leopard, though. The other two issues don't say which version of MacOSX they're using, but from the images they're clearly later than Snow Leopard.

I don't think it's due to lack of memory. The memory statistics on the F3 display show only about half the allocated memory being used.

Some more data points: It also occurs with these versions, using java 6 or java 7:
CoFHCore-[1.7.10]3.0.1-254.jar
ThermalExpansion-[1.7.10]4.0.0-176.jar
ThermalFoundation-[1.7.10]1.0.0-81.jar

It doesn't occur with these:
CoFHCore-[1.7.10]3.0.0B6-32.jar
ThermalExpansion-[1.7.10]4.0.0B5-13.jar
ThermalFoundation-[1.7.10]1.0.0B3-8.jar

So something changed between the beta and final releases of those mods.

One other thing I've discovered: Pressing F3-T to reload textures cures the problem.
